src/lib/components/schedule/schedule-recent-runs.svelte

```diff
@@ -30,7 +36,21 @@
 </script>
 
 <Panel class="w-full">
-  <h2 class="mb-4 text-2xl">{translate('schedules.recent-runs')}</h2>
+  <div class="flex justify-between">
+    <h2 class="mb-4 text-2xl">{translate('schedules.recent-runs')}</h2>
+    <Link
+      on:click={() => {
+        goto(
+          routeForWorkflowsWithQuery({
+            namespace,
+            query: workflowQuery,
+          }),
+        );
+      }}
+    >
+      View All Runs
+    </Link>
```

Comment:
Why not use `href` here? Also needs an i18n string

Reply:
👍 Cuz href does a hard reload all the apis instead of client-side routing